documentation/api/notation.rst

```diff
@@ -162,6 +162,67 @@ For version 1, 2 and 3 of the API, only equidistant timeseries data is expected
 - "duration" should also be a multiple of the sensor resolution.
 
 
+.. _describing_flexibility:
+
+Describing flexibility
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To compute a schedule, FlexMeasures needs to know state and context information which goes beyond
+the usual time series of the asset.
+
+We've grouped these two sources of information in two:
+
+Flex model
+""""""""""""
+
+The flexibility model describes to the scheduler what the flexible asset's state is,
+and what constraints or preferences should be taken into account.
+
+Which specific flexibility model is relevant in your scheduler usually relates to the sensor's asset type.
+Usually, not the whole flexibility model is needed (e.g. to be sent through the endpoint).
```

Comment:
For readability, I'd move the example (about not needing to send the whole model with each API call) to just after the explanation about how FlexMeasures tries to infer missing values (the next line).
```
This means that API and CLI users don't have to send the whole flex model every time.
```